implementations/rust/ockam/ockam_app_lib/src/shared_service/relay/create.rs

```diff
@@ -47,28 +63,26 @@ impl AppState {
         context: &Context,
         cli_state: &CliState,
         node_manager: Arc<InMemoryNode>,
-    ) -> Result<Option<RelayInfo>> {
+    ) -> Result<()> {
         trace!("Creating relay");
-        if !cli_state.is_enrolled().unwrap_or(false) {
-            trace!("Not enrolled, skipping relay creation");
-            return Ok(None);
-        }
         match cli_state.projects.default() {
             Ok(project) => {
-                if let Some(relay) = get_relay(node_manager.clone(), project.config()).await {
+                if let Some(_relay) = get_relay(&node_manager, cli_state).await? {
                     debug!(project = %project.name(), "Relay already exists");
                     self.update_orchestrator_status(OrchestratorStatus::Connected);
                     self.publish_state().await;
-                    Ok(Some(relay.clone()))
+                    Ok(())
                 } else {
+                    self.update_orchestrator_status(OrchestratorStatus::Connecting);
+                    self.publish_state().await;
                     debug!(project = %project.name(), "Creating relay at project");
                     let project_route = format!("/project/{}", project.name());
                     let project_address = MultiAddr::from_str(&project_route).into_diagnostic()?;
                     let relay = node_manager
                         .create_relay(
                             context,
                             &project_address,
-                            Some(relay_alias(project.config())),
+                            Some(relay_name(cli_state)?),
```

Comment:
Just to double-check, the full relay name is `forward_to_{alias}`, but when you call the `create_relay` function you must pass the `alias` only, without the `forward_to` prefix.

Perhaps the function is stripping the prefix just in case?

Reply:
I checked the logs and we are creating relays with the following name: `key=relay-forward_to_forward_to_Ibf5f10fc2add4dc091cf194813ccf32173aa5715`